app/views/admin/loans/questionnaires/_breakeven_table_question.html.slim

```diff
@@ -1,28 +1,33 @@
-.table-responsive.editable-tables
+.table-responsive.editable-tables.breakeven-tables
 
   .table-container
     h4.title = t("loan.breakeven.products", count: 2)
 
-    table.table.table-condensed.table-striped.table-bordered.editable-table data-table="products"
+    table.table.table-condensed.table-striped.table-bordered.editable-breakeven-table data-table="products"
```

Comment:
That part makes sense, but the problem is that I don't want the EditableView to also attach to this table. With that solution I would end up with two event handlers for `addRow`.